.bit/responses/4.6-Week 4 Step 6.md

```diff
@@ -0,0 +1,22 @@
+---
+files: n/a
+stepType: checks
+scripts: 4.6.spec.js
+week: 4
+step: 6
+name: Week 4 Step 6
+---
+
+Week 4 Step 6 ⬤⬤⬤⬤⬤⬤◯◯◯ | 🕐 Estimated completion: 5-20 minutes
+
+## One Cat isn't enough ~ Meow
+
+Demo: [EMILY GO INSERT A DEMO HERE]
+
+### ✅  Task:
+- [ ] Create an async function `getCatPic(name)` in `twocatz.js` that takes the input `name` and outputs the base 64 of the cat picture with the `name` on it produced by the [Cataas API](https://cataas.com/)
```

Comment:
Maybe the description is different on the google doc, but this is what it's supposed to be:

```js
function y1k3s() {
  if (document.getElementById("name").value != '') {
      let cat = document.getElementById("name").value
      document.getElementById("coffee").src = "https://cataas.com/cat/says/" + cat
  }
}
```
There is no getCatPic function or base64 involved